### PR TITLE
Adding step-level notifications information

### PR DESCRIPTION
**Pull request by karensawrey**
Iteration #2 because I messed up the task ID.

### CODE REVIEW

File: pages/pipelines/notifications.md.erb [hunk 1]
```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
...
+* On build status events in the Buildkite UI by using your Slack Notification Service's 'Build State Filtering' settings.
+* On step status and other non-build events by extending the Slack Notification Service using the `notify` attribute to your `pipeline.yml`.
 
-Before adding a `notify` attribute to your `pipeline.yml`, ensure an organization admin has set up a [Slack integration](/docs/integrations/slack) for the channel or user that you want to post to. For detailed information about setting up a Notification Service, see the [Slack integration page](/docs/integrations/slack).
+Before adding a `notify` attribute to your `pipeline.yml`, ensure an organization admin has set up a [Slack integration](/docs/integrations/slack) for the channel or user that you want to post to. Buildkite Enterprise users can also check the ['Manage Notifications Services'](https://buildkite.com/organizations/~/member-permissions) checkbox to create, edit, or delete notification services. For detailed information about setting up a Notification Service, see the [Slack integration page](/docs/integrations/slack).
```
Comment (pzeballos) [inline pages/pipelines/notifications.md.erb:107]: I would also add that if they rename the channel or make a modification (like changing it from public to private), they need to set up a new integration.

**Comment by pzeballos** [comment]
The rest looks good!

File: pages/pipelines/notifications.md.erb (hunk 1 above)
Comment (karensawrey) [reply pages/pipelines/notifications.md.erb:107]: Thank you so much for your review, @pzeballos!

File: pages/pipelines/notifications.md.erb [hunk 2]
```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
...
 
-## Slack Channel and Direct Messages
+## Slack channel and direct messages
 
-If you need fine-grained control over your notifications, or want to send notifications triggered by non-build events, the [Slack Notification Service](/docs/integrations/slack) can be extended using the `notify` attribute.
+You can set notifications:
 
-If you only need to send notifications on build status events, you can set this up in Buildkite using your [Slack Notification Service's](/docs/integrations/slack) 'Build State Filtering' settings.
+* On build status events in the Buildkite UI by using your Slack Notification Service's 'Build State Filtering' settings.
+* On step status and other non-build events by extending the Slack Notification Service using the `notify` attribute to your `pipeline.yml`.
```
Comment (plaindocs) [inline pages/pipelines/notifications.md.erb:105]:
```suggestion
* On step status and other non-build events, by extending the Slack Notification Service using the `notify` attribute in your `pipeline.yml`.
```

File: pages/pipelines/notifications.md.erb [hunk 3]
```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
...
 * `build skipped`
 
-## Slack Channel and Direct Messages
+## Slack channel and direct messages
 
-If you need fine-grained control over your notifications, or want to send notifications triggered by non-build events, the [Slack Notification Service](/docs/integrations/slack) can be extended using the `notify` attribute.
+You can set notifications:
 
-If you only need to send notifications on build status events, you can set this up in Buildkite using your [Slack Notification Service's](/docs/integrations/slack) 'Build State Filtering' settings.
+* On build status events in the Buildkite UI by using your Slack Notification Service's 'Build State Filtering' settings.
```
Comment (plaindocs) [inline pages/pipelines/notifications.md.erb:104]:
```suggestion
* On build status events in the Buildkite UI, by using your Slack Notification Service's 'Build State Filtering' settings.
```

File: pages/pipelines/notifications.md.erb [hunk 4]
```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
...
+Once a Slack channel has been configured in your organization, add a Slack notification to your pipeline using the `slack` attribute of the `notify` YAML block. Remember that if you rename or modify the Slack channel for which the integration was set up, for example if you change it from public to private, you need to set up a new integration.
 
 <section class="Docs__troubleshooting-note">
   <p>When using just a channel name, you must specify it in quotes, as otherwise the <code>#</code> will cause the channel name to be treated as a comment.</p>
 </section>
 
-For example, to deliver notifications to the `#general` channel of all configured workspaces:
+For example, to deliver build-level notifications to the `#general` channel of all configured workspaces:
 
 ```
```
Comment (plaindocs) [inline pages/pipelines/notifications.md.erb:117]:
```suggestion
```yaml
```

File: pages/pipelines/notifications.md.erb [hunk 5]
```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
...
 notify:
   - slack: "#general"
 ```
 {: codeblock-file="pipeline.yml"}
 
-For example, to deliver notifications to user `@someuser` in all configured workspaces:
+For step-level notifications to the `#general` channel of all configured workspaces, use the following syntax:
+
+```
```
Comment (plaindocs) [inline pages/pipelines/notifications.md.erb:125]:
```suggestion
```yaml
```

File: pages/pipelines/notifications.md.erb [hunk 6]
```diff
@@ -97,39 +97,63 @@ Basecamp notifications happen at the following [events](/docs/apis/webhooks#even
...
 notify:
   - slack: "@someuser"
 ```
 {: codeblock-file="pipeline.yml"}
 
-To send a notification to one particular workspace and channel or workspace and user, specify the workspace name as well:
+For step-level notifications to to user `@someuser` in all configured workspaces:
 
-For channels
+```
```
Comment (plaindocs) [inline pages/pipelines/notifications.md.erb:144]:
```suggestion
```yaml
```

**Review by plaindocs — APPROVED** [review]
LGTM

**Pull request merged**